### PR TITLE
feat: loading indicator

### LINKED ISSUE #1119: LoadingIndicator widget
**Enhancement**

Loading indicator widget

Package Version: latest

**Code**

```jsx
<LoadingIndicator />
```

**Expected behavior:**

An indeterminate loading indicator should render.

**Actual behavior:**

No loading indicator exists yet.

### PR DESCRIPTION
**Pull request by bitpshr**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/framework/blob/master/STYLE.md)
* [x] Unit tests are included in the PR
* [x] For new widgets, an entry has been added to the `.dojorc`
* [x] Any widget variant uses `theme.compose` like [this](https://github.com/dojo/widgets/issues/847)
* [x] WidgetProperties are exported

**Description:**

This pull request adds an indeterminate loading indicator widget.

Resolves #1119

**Preview**

preview

### CODE REVIEW

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-2m71omyyi.now.sh
* Tests Deployment: https://dojowidgets-r2m2j17t8.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-2m71omyyi.now.sh
* Tests Deployment: https://dojowidgets-r2m2j17t8.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-2m71omyyi.now.sh
* Tests Deployment: https://dojowidgets-6hqafyaqd.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-kazvaemr7.now.sh
* Tests Deployment: https://dojowidgets-akk0krv92.now.sh

File: src/theme/material/loading-indicator.m.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.root {
+	composes: mdc-linear-progress from '@material/linear-progress/dist/mdc.linear-progress.css';
+	composes: mdc-linear-progress--indeterminate from '@material/linear-progress/dist/mdc.linear-progress.css';
```
Comment (tomdye) [inline src/theme/material/loading-indicator.m.css]:
```suggestion
	composes: mdc-linear-progress mdc-linear-progress--indeterminate from '@material/linear-progress/dist/mdc.linear-progress.css';
```

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-fv1c5mke5.now.sh
* Tests Deployment: https://dojowidgets-dzrmgid3l.now.sh

**Pull request merged**